tests/observability/metrics/conftest.py

```diff
@@ -899,3 +899,60 @@ def virt_handler_pods_count(hco_namespace):
             namespace=hco_namespace.name,
         ).instance.status.numberReady
     )
+
+
+@pytest.fixture(scope="class")
+def nodes_allocatable_info(nodes):
+    selected_node = nodes[0]
+    selected_node_instance = selected_node.instance
```

Comment:
Should be a separate fixture. For allocable data create a fixture to send back selected_node_instance.status.allocatable and for labels another fixture should send back selected_node_instance.metadata.labels